experimental/ner_wikiner_speedster/requirements.txt

```diff
@@ -0,0 +1,4 @@
+nebullvm>=0.6
+speedster>=0.0.1
```

Comment:
Question: do we still need to specify `nebullvm` in the requirements, or is it going to be pulled by `speedster anyway?